README.md

```diff
@@ -431,11 +431,10 @@ bundle exec rubocop # Runs the lint (use `--fix` for autocorrect)
 
 ## Releasing a new gem version
 
-1. Execute `bundle exec rake bump:minor`, which bumps the version in my local machine in a specific commit
-2. Change the CHANGELOG and amend that previous commit, so the bump commit and the CHANGELOG changes are in the same commit (example)
-3. Push to Github
+1. Execute `bundle exec rake bump:patch`, or minor or major according to [SemVer](https://semver.org), this bumps the version in my local machine in a specific commit
+2. Update the CHANGELOG amending the previous commit, so the bump commit and the CHANGELOG changes are in the same commit
+3. Push to Github, raise a PR, get it approved and merge
 4. Post a message in Slack `#rest-api`, so advocacy are aware that we are going to release a new gem, just in case any customer complains about something related to the gem
-5. After a couple of hours, run `bundle exec rake release`, which automatically pushes the tag to GitHub and releases a new version to Rubygems (it does the gem push for you)
```

Comment:
This line was already outdated.

PS: I'm not sure we need to keep the CHANGELOG in the same commit as the bump. It's the last opportunity to check the changelog is correct before releasing the gem, but personally, I would prefer if we were to update it as we develop our features.